kansha/card/fts_schema.py

```diff
@@ -12,27 +12,25 @@
 """
 
 from kansha.services.search import schema
+from kansha.services.components_repository import CardExtensions
 
 
 class Card(schema.Document):
     title = schema.TEXT(stored=True)
-    description = schema.TEXT
-    comments = schema.TEXT
-    tags = schema.TEXT
-    lists = schema.TEXT
     board_id = schema.INT
     archived = schema.BOOLEAN
 
     @classmethod
+    def update_schema(cls, card_extensions):
+        for name, extension in card_extensions.items():
+            field = extension.get_schema_def()
+            if field is not None:
+                cls.fields[name] = field
+
+    @classmethod
     def from_model(cls, card):
```

Comment:
Card extensions can provide some more fields to be indexed (see Checklists, Comments and Description for some example). This updates the schema definition.